src/main/java/org/checkerframework/specimin/TargetMethodFinderVisitor.java

```diff
@@ -183,10 +184,9 @@ public Visitable visit(VariableDeclarator node, Void arg) {
   @Override
   public Visitable visit(MethodDeclaration method, Void p) {
     String methodDeclAsString = method.getDeclarationAsString(false, false, false);
-    // The substring here is to remove the method's return type. Return types cannot contain spaces.
+    // The substring here is to remove the method's return type.
```

Comment:
This substring has been moved into `removeMethodReturnType`, so this comment doesn't make sense anymore.